ambry-utils/src/main/java/com.github.ambry.utils/Throttler.java

```diff
@@ -90,7 +90,7 @@ public void maybeThrottle(double observed) throws InterruptedException {
   /**
    * Disable the throttler for good.
    */
-  public void close() {
+  public void disable() {
```

Comment:
No there aren't any - it looks like the function's behavior cannot be tested from the outside.